plugins/removeAttrs.js

```diff
@@ -107,10 +111,13 @@ exports.fn = function(item, params) {
                     // loop attributes
                 item.eachAttr(function(attr) {
                     var name = attr.name;
+                    var value = attr.value;
 
+                    if(!(preserveCurrentColor && value === 'currentColor')) {
```

Comment:
only potentially remove if we have NOT enabled this option AND the value is not currentColor. This is essentially same as what we did for svgstore: https://github.com/FWeinb/grunt-svgstore/pull/63/files#diff-239be4703ce423ab0e43053cdf58384bR176